packages/preview/touying-unistra-pristine/1.0.0/typst.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+name = "touying-unistra-pristine"
+version = "1.0.0"
+entrypoint = "src/lib.typ"
+authors = [
+  "Enzo Doyen <https://edoyen.com/>"
+]
+license = "MIT"
+description = "Touying theme adhering to the core principles of the style guide of the University of Strasbourg, France"
+# homepage = ""
+repository = "https://github.com/spidersouris/touying-unistra-pristine"
+keywords = ["touying", "unistra", "strasbourg", "presentation"]
+categories = ["presentation"]
+disciplines = []
+compiler = "0.11.0"
+exclude = []
+
+[template]
```

Comment:
The template will be copied to the user's project and must be self-contained. The relative path referencing the cat will not work as-is. Typically, the directory is also called `template` rather than `example`.